sites/partners/src/listings/PaperListingDetails/sections/DetailRankingsAndResults.tsx

```diff
@@ -70,13 +71,13 @@ const DetailRankingsAndResults = () => {
       {listing.waitlistMaxSize && (
         <GridSection columns={3}>
           <ViewItem label={t("listings.waitlist.maxSize")}>
-            {listing.waitlistMaxSize.toString()}
+            {getDetailFieldNumber(listing.waitlistMaxSize)}
```

Comment:
Separate issue but was preventing me from loading the LM page to test this and was a quick fix